src/main/java/com/orbitz/consul/model/health/Service.java

```diff
@@ -17,21 +18,24 @@
 public abstract class Service {
 
     @JsonProperty("ID")
-    public abstract  String getId();
+    public abstract String getId();
```

Comment:
Unrelated change - removed an extraneous space. Same change a few other places in this file.